scripts/mirror.js

```diff
@@ -219,71 +219,61 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
 };
 
 /**
- * @param {SupportStatement} originalData
- * @param {SupportStatement} sourceData
- * @param {string} source
+ * @param {SupportStatement} comp
  * @returns {SupportStatement}
  */
-const bumpEdge = (originalData, sourceData, source) => {
-  let newData = {};
-
-  if (source == 'ie') {
-    if (sourceData.version_removed && sourceData.version_removed !== null) {
-      newData.version_added = false;
-    } else if (sourceData.version_added !== null) {
-      newData.version_added = sourceData.version_added ? '12' : null;
-    }
+const bumpEdge = comp => {
+  let newData = comp['edge'];
+  let originalData = comp['edge'];
+  let ieData = comp['ie'];
+  let chromeData = comp['chrome'];
 
-    if (sourceData.notes) {
-      newData.notes = updateNotes(
-        sourceData.notes,
-        /Internet Explorer/g,
-        'Edge',
-      );
-    }
-  } else if (source == 'chrome') {
-    newData = originalData == undefined ? sourceData : originalData;
+  if (ieData.version_removed && ieData.version_removed !== null) {
+    newData.version_added = false;
+  } else if (ieData.version_added !== null) {
+    newData.version_added = ieData.version_added ? '12' : null;
+  }
 
-    let chromeFalse =
-      sourceData.version_added === false ||
-      sourceData.version_removed !== undefined;
-    let chromeNull = sourceData.version_added === null;
+  let chromeFalse =
+    chromeData.version_added === false ||
+    chromeData.version_removed !== undefined;
```

Comment:
This bit makes me wonder what happens for a feature behind a flag. This script doesn't seem to handle flags at all. The data for Edge seems to mostly have flags copied, but it seems to me there's still a but waiting here: if the feature was added behind a flag in Chrome, `chromeFalse` will still end up false here, and we'll mark it as supported in Edge. I've verified this by modifying Bluetooth.json locally.

I think we'll need a step where support statements are pre-processed to apply flags, alternative_name, and perhaps more.

It's fine to do that in a follow-up though, it's a generic problem, not specific to mirroring Edge.

Reply:
I think we should do this in a follow-up, considering that it's a problem that applies to all browsers.

Reply:
Yep! Can you file an issue for that?

Reply:
Filed as #6625!